news/2021-08-15-summer-2021-tf-fanart-voting-open.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: "Summer 2021 Track and Field Fanart Contest Voting Open!"
+date: 2021-08-15 12:00:00 +0000
+---
+
+**80** artistic contenders are at the starting line and ready to race, now all we need are your votes!
+
+![](https://assets.ppy.sh/contests/133/header.jpg)
+
+In a classic display of the talent simply brimming in the osu! community, a horde of awesome artists have now passed the baton to the rest of us, and it is time to choose only **10** of these amazing entries to receive your votes.
+
+As is tradition, we've assembled a brief slideshow of all qualifying entries set to [**Redside**](https://osu.ppy.sh/beatmaps/artists/184)'s Aspiration to help get people moving:
+
+<div align="center">
+    <video width="90%" controls>
+        <source src="https://assets.ppy.sh/contests/133/summer2021.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+[**Head on over to the Summer 2021 Track and Field Contest listing to vote now!**](https://osu.ppy.sh/community/contests/133)
+
+The voting will remain open for **7 days**, so you better get in quick before the race is over.
+
+A choice selection of the top voted entries (usually the top 10 or 15, but it could be more or less) will be this year's Summer seasonal backgrounds and also nab their artists some neat prizes, too.
```

Comment:
```suggestion
A selection of the top voted entries (usually the top 10 or 15, but it could be more or less) will be this year's Summer seasonal backgrounds and also nab their artists some neat prizes, too.
```